docs/DeveloperGuide.md

```diff
@@ -283,32 +283,165 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BobTheBodyBuilder` and the **Actor** is the `Gym Manager`, unless specified otherwise)
```

Comment:
Are you able to add the acronym for BobTheBodyBuilder and reuse this for all the use cases below. I think it'll make the use case shorter and easier to read
```suggestion
(For all use cases below, the **System** is the `BobTheBodyBuilder (BTBB)` and the **Actor** is the `Gym Manager`, unless specified otherwise)
```

Reply:
I just added BTBB to the use cases.